nvm.sh

```diff
@@ -859,7 +859,6 @@ nvm_get_arch() {
   case "$NVM_UNAME" in
     *x86_64*) NVM_ARCH=x64 ;;
     *i*86*) NVM_ARCH=x86 ;;
-    *armv6l*) NVM_ARCH=arm-pi ;;
```

Comment:
I concur with you. That is the exact logic i'm thinking: 
- if `node`, then look for `arm-pi`
- if `io.js`, then look for `uname -m`

I will revise my pull request by this weekend!

I also agree with the error - maybe I can add that in another separate pull request just to be clean. For reference, the current error message in a raspberry pi is as follows:

```
$ nvm install 0.10.36
######################################################################## 100.0%
Checksums empty

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Binary download failed, trying source.
```